src/test/java/org/assertj/core/api/file/FileAssert_hasFileName_Test.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.api.file;
+
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.FileAssert;
+import org.assertj.core.api.FileAssertBaseTest;
+import org.junit.jupiter.api.DisplayName;
+
+@DisplayName("FileAssert hasFileName")
```

Comment:
This is not nessesary as the name generator is used
https://github.com/assertj/assertj-core/blob/b45e499446d3296dd11c480b0bd2087328989f05/src/test/resources/junit-platform.properties#L1

Reply:
Was not aware of that. Thank you 😄 . Then the display name will be `FileAssert hasFileName Test` instead of `FileAssert hasFileName`. 

Would that be fine? as the [example in contributing guide](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md) has display name without `Test`

Reply:
Ideed the guide says that your code is good. I will have to revisit it .... 😄

Reply:
The correct display name is indeed `FileAssert hasFileName`.

@joel-costigliola thinking aloud, what if we had our own custom implementation of `DisplayNameGenerator`, which removes underscores and also the suffix `Test`? This way we might avoid the repetitive `@DisplayName` on every test.

Reply:
With e8e221dafb69c88b64bddb8accd497e26c1635ee it is no longer necessary to add a `@DisplayName` on the tests. @ShivakumarSwamy feel free to rebase this PR.

Reply:
Cool 🎉 
Removed display name, updated minor version in since tag and rebased.